libs/ardour/location.cc

```diff
@@ -1611,11 +1611,13 @@ Locations::marks_either_side (timepos_t const & pos, timepos_t& before, timepos_
 
 	positions.sort ();
 
-	std::list<timepos_t>::iterator i = positions.begin ();
-
-	while (i != positions.end () && *i < pos) {
-		++i;
-	}
+	std::list<timepos_t>::iterator i = std::find_if(
```

Comment:
```suggestion
	auto i = std::find_if(
```
everywhere.

Reply:
Let me know if you think it's OK now for you.